### PR TITLE
[CHF-477] Health Check implementation for Z-Wave Dimmer Switch Generic

### PR DESCRIPTION
**Pull request by pchomal**
Health Check implementation for Z-Wave Dimmer Switch Generic concerning below devices:

1. Leviton 1000W Incandescent Dimmer
2. Remotec Plug-In Dimmer (ZDS-110)

### CODE REVIEW

**Comment by pchomal** [comment]
@jackchi @ShunmugaSundar 

Please review and merge.

**Comment by pchomal** [comment]
@jackchi 
Is this PR good to go?
Have just added the fingerprints (using WWST names), and not any new capabilities.

File: devicetypes/smartthings/zwave-dimmer-switch-generic.src/zwave-dimmer-switch-generic.groovy [hunk 1]
```diff
@@ -28,6 +28,8 @@ metadata {
 		fingerprint mfr:"001D", prod:"1001", model:"0334", deviceJoinName: "Leviton 3-Speed Fan Controller"
 		fingerprint mfr:"001D", prod:"0602", model:"0334", deviceJoinName: "Leviton Magnetic Low Voltage Dimmer"
 		fingerprint mfr:"001D", prod:"0401", model:"0334", deviceJoinName: "Leviton 600W Incandescent Dimmer"
+		fingerprint mfr:"0111", prod:"8200", model:"0200", deviceJoinName: "Remotec Technology Plug-In Dimmer"
+		fingerprint mfr:"1104", prod:"001D", model:"0501", deviceJoinName: "Leviton 1000W Incandescant Dimmer Dimmer"
```
Comment (jackchi) [inline devicetypes/smartthings/zwave-dimmer-switch-generic.src/zwave-dimmer-switch-generic.groovy]: @pchomal Is this suppose to be `Dimmer Dimmer`?

**Review by jackchi — COMMENTED** [review]
Other than that, it looks good 👍 1

File: devicetypes/smartthings/zwave-dimmer-switch-generic.src/zwave-dimmer-switch-generic.groovy (hunk 1 above)
Comment (pchomal) [reply devicetypes/smartthings/zwave-dimmer-switch-generic.src/zwave-dimmer-switch-generic.groovy]: @jackchi Good to go now?

**Comment by jackchi** [comment]
@mckeed @tylerlange @workingmonk - These devices appear on the WWST page but so far they don't have individual names. 
Does this look okay to merge?

File: devicetypes/smartthings/zwave-dimmer-switch-generic.src/zwave-dimmer-switch-generic.groovy (hunk 1 above)
Comment (pchomal) [reply devicetypes/smartthings/zwave-dimmer-switch-generic.src/zwave-dimmer-switch-generic.groovy]: @jackchi Updated the Readme as well based on my changes.

**Comment by tylerlange** [comment]
I don't have the ability to double check this right now, but the names themselves look good.

**Comment by mckeed** [comment]
:+1:

**Pull request merged**